tools/marvin/marvin/config/test_data.py

```diff
@@ -2186,6 +2371,33 @@
                 "Dns": "VpcVirtualRouter"
             }
         },
+        "vpc_offering_configdrive_withoutdns": {
```

Comment:
General note wrt changes in `test_data.py` - please move to specific marvin test.

Reply:
@rhtyd i created CLOUDSTACK-10219 to track this. We will take this forward. I expect this not to happen for 4.11 probably though. Hope that's ok.

Reply:
@krissterckx can you please get them moved to a specific test. Thanks.